pkgs/by-name/de/delugia-code/package.nix

```
@@ -0,0 +1,23 @@
+{
+  lib,
+  stdenvNoCC,
+  fetchzip,
+}:
+stdenvNoCC.mkDerivation rec {
+  pname = "delugia-code";
+  version = "v2404.23";
+  src = fetchzip {
+    url = "https://github.com/adam7/delugia-code/releases/download/${version}/delugia-complete.zip";
+    hash = "sha256-Zth2D0Cp6T8Ri+HK9c5JEKYF9OiwHS3EqKiulaKMMio=";
+  };
+  installPhase = ''
+    install -m444 -Dt $out/share/fonts *.ttf
+  '';
+  meta = with lib; {
```

```suggestion
  meta = {
```
remove with lib; as it increasese eval times.